doc/jobs/doc_jobs.rst

```diff
@@ -31,6 +31,12 @@ Each *doc build files* can have one of three specific documentation types:
   honor an environment variable identifying the destination of the generated
   documentation.
 
+* the **external_site** type also operates on a single independent repository.
+  For each *doc build file* with that type a Jenkins job is created. Such repository
```

Comment:
Each sentence should start on a new line (see the [developer guideline](https://github.com/ros2/ros2/wiki/Developer-Guide#markdown)).

Same below.

Reply:
Good catch. Fixed.